provider/resource.go

```diff
@@ -1760,7 +1762,6 @@ func Provider() tfbridge.ProviderInfo {
 			},
 
 			// Network
-			"azurerm_network_function_azure_traffic_collector": {Tok: azureResource(azureNetwork, "FunctionAzureTrafficController")},
```

Comment:
Based on the upstream docs, looks like this new resource belongs in a new module `network_function`, not `network`